CHANGELOG-v3.adoc

```diff
@@ -5,6 +5,38 @@ For upgrade instructions, please refer to https://docs.gravitee.io/apim/3.x/apim
 *Important:* If you plan to skip versions when you upgrade, ensure that you read the version-specific upgrade notes for each intermediate version. You may be required to perform manual actions as part of the upgrade.
 
 
+== https://github.com/gravitee-io/issues/milestone/577?closed=1[APIM - 3.17.5 (2022-08-01)]
+
+=== Bug fixes
+
+*_Console_*
+
+- Closed subscription is displayed as still subscribed https://github.com/gravitee-io/issues/issues/7879[#7879]
+- Debug mode join multi value headers https://github.com/gravitee-io/issues/issues/7809[#7809]
+
+*_General_*
+
+- Log file are not created https://github.com/gravitee-io/issues/issues/8066[#8066]
+- Merge 3.16.4 in 3.17.x https://github.com/gravitee-io/issues/issues/7869[#7869]
+
+*_Management_*
+
+- Update a role API requires an `id` field when this is not necessary/required based on the doc https://github.com/gravitee-io/issues/issues/7661[#7661]
+
+*_Policies_*
+
+- Groovy policy error on HTTP headers https://github.com/gravitee-io/issues/issues/7810[#7810]
+- Truncated `request.content` in Callout Http Policy when body contains accents https://github.com/gravitee-io/issues/issues/8109[#8109]
+
+=== Features
+
+*_Node_*
+
+- Expose endpoints for retrieving heap dump and thread dump https://github.com/gravitee-io/issues/issues/8222[#8222]
+
+ 
+ 
```

Comment:
Would be great to add a message to tell users this is the last 3.17

Reply:
Done